latch_cli/main.py

```diff
@@ -684,3 +684,37 @@ def test_data_ls():
     click.secho("Listing your managed objects by full S3 path.\n", fg="green")
     for o in objects:
         print(f"\ts3://latch-public/{o}")
+
+
+@main.command()
+@click.argument("srcs", nargs=-1)
+@click.argument("dst", nargs=1)
+@click.option(
+    "--delete",
+    help="Delete extraneous files from destination.",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--ignore-unsyncable",
+    help=(
+        "Synchronize even if some source paths are do not exist or refer to special"
```

Comment:
```suggestion
        "Synchronize even if some source paths do not exist or refer to special"
```

what is a special file